Signal/src/contact/OWSContactsManager.m

```diff
@@ -523,17 +523,29 @@ - (NSAttributedString *_Nonnull)formattedFullNameForContact:(Contact *)contact f
     NSAttributedString *_Nullable firstName, *_Nullable lastName;
     if (ABPersonGetSortOrdering() == kABPersonSortByFirstName) {
         if (contact.firstName) {
-            firstName = [[NSAttributedString alloc] initWithString:contact.firstName attributes:boldFontAttributes];
+            firstName = [[NSAttributedString alloc]
+                initWithString:[contact.firstName
```

Comment:
take it or leave it, but should we do this in the contact class? We're never going to want whitespace at the beginning/end of a name are we?